rhombus/scribblings/ref-stxobj-meta.scrbl

```diff
@@ -14,8 +14,8 @@
                         at_stx :: Syntax)
     :: None
   fun syntax_meta.error(message :: ReadableString,
-                        at_stx :: Syntax,
-                        in_stx :: Syntax || List.of(Syntax))
+                        in_stx :: Syntax,
+                        at_stx :: Syntax || List.of(Syntax))
```

Comment:
This shouldn’t be included, I think.  (I see that there’s some problem in this doc, but I’ll fix it in a separate commit.)

Reply:
Removed.